src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/upgrade/upgrade.component.html

```diff
@@ -21,68 +57,74 @@
              [ngStyle]="healthData.health.status | healthColor"
              [ngbPopover]="healthChecks"
              popoverClass="info-card-popover-cluster-status">
-             {{ healthData.health.status | healthLabel | uppercase }}
+            {{ healthData.health.status | healthLabel | uppercase }}
           <i *ngIf="healthData.health?.status !== 'HEALTH_OK'"
              class="fa fa-exclamation-triangle"></i>
         </div>
       </div>
-      <div class="col-lg-3 h-50 d-flex flex-column border justify-content-center align-items-center">
-        <span class="bold">Upgrade Status</span>
-        <ng-container *ngIf="upgradeInfo.versions.length > 0; else noUpgradesAvailable">
-          <span class="mt-2"
-                i18n>
-          <i [ngClass]="[icons.up]"
-             class="text-info"></i>
-            Upgrade available</span>
-          <div i18n-ngbTooltip
-               [ngbTooltip]="(healthData.mgr_map | mgrSummary).total <= 1 ? 'To upgrade, you need minimum 2 mgr daemons.' : ''">
-            <button class="btn btn-accent mt-2"
-                    id="upgrade"
-                    aria-label="Upgrade now"
-                    [disabled]="(healthData.mgr_map | mgrSummary).total <= 1"
-                    (click)="startUpgradeModal()"
-                    i18n>Upgrade now</button>
-          </div>
-        </ng-container>
-      </div>
-      <div class="col-lg-3 h-50 d-flex flex-column border justify-content-center align-items-center">
-        <span class="bold">MGR Count</span>
-        <span class="mt-1">
+    </cd-card>
+
+    <cd-card class="col-sm-3 px-3 d-flex"
+             cardTitle="MGR Count"
```

Comment:
In your screen-recording I don't see any "MGR Count" card anymore 🤔

Reply:
Yeah it was an outdated one! sorry. I updated the correct recording!